example/lib/develop/model/ble_peripheral.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:equatable/equatable.dart';
```

Comment:
why the code is placed in `develop` package/dir? what is the idea behind the name?

Reply:
Since this is still a draft PR, I wanted to still have the old implementation present alongside in the project if I needed a reference. The idea was to move it to the main package when the PR is in the "ready for review" state.